tkg/test/tkgpackageclient/package_plugin_integration_test.go

```diff
@@ -21,19 +21,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/util/retry"
 
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/test/framework/exec"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/test/framework/exec"
```

Comment:
Is it possible that this subdirectory ```tkg/test/tkgpackageclient``` remain where it is? I'm working to moving it to its own subdirectory and this change will cause big merge conflict